packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -186,6 +183,10 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
     this._pageCache = {};
   }
 
+  public get PageRefs(): Readonly<Record<string, unknown>> {
```

Comment:
What's the reasoning for having this pascal case?

Reply:
Nothing specific; was just think it should be as a public property. I'll change it.